docs/DeveloperGuide.md

```diff
@@ -390,6 +395,91 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
         Use case resumes at step 2. 
 
+**Use case: Add a sale to a contact**
+
+**MSS**
+
+1.  User requests to list contacts
+2.  StonksBook shows a list of contacts
+3.  User requests to add a sale to a specific contact in the list
+4.  StonksBook adds a sale to the specific contact
```

Comment:
Should we be consistent with whether to put a full stop after each step in the use cases? I think the DG will look cleaner that way.